packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/exception/CodedException.kt

```diff
@@ -243,3 +244,7 @@ class JavaScriptEvaluateException(
 internal class UnsupportedClass(
   clazz: KClass<*>,
 ) : CodedException(message = "Unsupported type: '$clazz'")
+
+internal class PromiseWasAlreadyResolved(functionName: String) : CodedException(
+  message = "Promised pass to '$functionName' was already resolved. It will lead to a crash in the production environment!"
+)
```

Comment:
What about this 👇 ?
```suggestion
internal class PromiseAlreadySettledException(functionName: String) : CodedException(
  message = "Promised pass to '$functionName' was already settled. It will lead to a crash in the production environment!"
)
```